ClientTests/Home/FirefoxHomeViewModelTests.swift

```diff
@@ -18,7 +18,7 @@ class FirefoxHomeViewModelTests: XCTestCase {
         XCTAssertEqual(viewModel.shownSections.count, 2)
     }
 
-    func testNumberOfSection_updatingData_adds2Sections() {
+    func testNumberOfSection_updatingData_adds4Sections() {
```

Comment:
this test is failing for me intermittent, 2 sections are loading first and then the other 4 reload